doc/se/Snabbstart

```diff
@@ -40,32 +40,32 @@ För de otåliga följer här instruktioner för att köra igång ScummVM i fem
 3. Starta ScummVM.
 
 Om programmet nu visas på engelska istället för svenska, gör såhär för att byta språk:
-- Klicka på Options.
-- Klicka på högerpilen i tab-raden och navigera till Misc-tabben.
-- Välj Svenska i GUI Language-menyn och klicka på OK.
-- Konfirmera meddelandet som visas, klicka på Quit för att avsluta ScummVM och starta sedan om programmet.
+- Klicka på ”Options”.
+- Klicka på högerpilen i tab-raden och navigera till “Misc”-tabben.
```

Comment:
Like here?

Reply:
Opening quotation marks (“) are not used in standard Swedish typography, so the opening quotation mark before Misc should be changed into a closing one. As an aside, most Swedish style guides would not recommend using quotation marks inside a word like that, so in this case the preferred form would probably be something like "Misctabben." Also, the most common translation of "tab" is "flik".